Custom Device Source/Custom Device Scan Engine.xml

```diff
@@ -1390,4 +1390,4 @@
   <CustomXML>
     <Dummy />
   </CustomXML>
-</CustomDevice>
\ No newline at end of file
+</CustomDevice>
```

Comment:
Not sure what changed about this line; I made the change to the file in the GitHub editor directly.

Reply:
I imagine it added a new line that wasn't there previously.